.github/workflows/release.yml

```diff
@@ -14,15 +14,15 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version-file: go.mod
           check-latest: true
           cache: true
       - name: Import GPG key
         id: import_gpg
-        uses: hashicorp/ghaction-import-gpg@v2.1.0
-        env:
-          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
-          PASSPHRASE: ${{ secrets.PASSPHRASE }}
+        uses: crazy-max/ghaction-import-gpg@111c56156bcc6918c056dbef52164cfa583dc549 # pin@v5
```

Comment:
docs show using crazy-max/ghaction-import-gpg@v5 - I take it that didn't work here for some reason?

Reply:
When we created this provider, they were recommending their own fork. I ran into an issue when trying to release a new version and saw they were recommending this one now